src/main/java/com/github/osisoft/ocs_sample_library_preview/sds/TypesClient.java

```diff
@@ -154,6 +171,22 @@ public String getTypes(String tenantId, String namespaceId, int skip, int count)
      * @throws SdsError any error that occurs
      */
     public String getTypes(String tenantId, String namespaceId, int skip, int count, String query) throws SdsError {
+        return getTypes(tenantId, namespaceId, skip, count, query, baseClient.getHttpHeadersForRequest());
+    }
+
+    /**
+     * gets stream types
+     * 
+     * @param tenantId    tenant to work against
+     * @param namespaceId namespace to work against
+     * @param skip        number of types to skip, useful in paging
+     * @param count       number of types to return
+     * @param query       query to reduce the number of types returned
+     * @param headers     http headers to use in request
+     * @return string of the types
+     * @throws SdsError any error that occurs
+     */
+    public String getTypes(String tenantId, String namespaceId, int skip, int count, String query, Map<String, String> headers) throws SdsError {
```

Comment:
does this work?

Reply:
that's my bad, I forgot that types aren't included in community calls. Thanks!